### PR TITLE
Fix spurious missing event handling with Deflect & Qt window system

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by eile** [comment]
+1

File: eq/qt/messagePump.h [hunk 1]
```diff
@@ -48,6 +50,9 @@ class MessagePump : public eq::MessagePump
     lunchbox::a_int32_t _wakeup;
     std::unordered_map< deflect::Proxy*,
                         std::unique_ptr< QSocketNotifier > > _notifiers;
+    std::unordered_map< deflect::Proxy*,
+                        QMetaObject::Connection > _connections;
+    QTimer* _timer;
```
Comment (rdumusc) [inline eq/qt/messagePump.h]: not a unique_ptr?

**Comment by tribal-tec** [comment]
Updated

File: eq/qt/messagePump.cpp [hunk 2]
```diff
@@ -31,6 +31,7 @@ namespace eq
 namespace qt
 {
 MessagePump::MessagePump()
+    : _timer( nullptr )
```
Comment (rdumusc) [inline eq/qt/messagePump.cpp]: not needed

**Comment by rdumusc** [comment]
+1

**Comment by tribal-tec** [comment]
retest this please

**Pull request merged**